conf/test_bc_nodx.config

```diff
@@ -22,6 +22,7 @@ params {
   flowcell = 'FLO-MIN106'
   kit = 'SQK-DCS108'
   skip_demultiplexing = true
+  skip_quantification = true
```

Comment:
Same here.

Reply:
The protocol is set to `DNA` for this test so we need to have it here to avoid the error below:

`Invalid protocol option if performing quantification: DNA. Valid options: 'cDNA', 'directRNA'`

Reply:
I can't test it locally so just removed them here https://github.com/nf-core/nanoseq/pull/89/commits/45b9d02f5e2371a84e816305de4305dafbe9fa40 to see if the tests still pass.

Reply:
This [samplesheet](https://raw.githubusercontent.com/nf-core/test-datasets/nanoseq/samplesheet_bc_nodx.csv) doesn't provide a GTF file so the test will fail without `-skip_quantification` which is expected behaviour.